GOVERNANCE.md

```diff
@@ -0,0 +1,139 @@
+Governance for the R data.table project
+
+# Purpose and scope
+
+## This document
+
+The purpose of this document is to define how people related to the project work together, so that the project can expand to handle a larger and more diverse group of contributors.
+
+## The R package
+
+The purpose of the project is to maintain the R data.table package, which is guided by the following principles:
+
+* No external Imports/LinkingTo/Depends dependencies (external meaning those not maintained by the project)
```

Comment:
Somewhat minor, I think order matters here (or we should explicitly say "in no particular order", and alphabetize). 

How about:

```suggestion
* Time & memory efficiency
* Concise syntax (minimal redundancy in code)
* No external Imports/LinkingTo/Depends dependencies (external meaning those not maintained by the project)
* Few (if any) Suggests/Enhances dependencies
* Stable code base (strong preference for user-friendly back-compatibility with data.table itself and with old versions of R)
* Comprehensive and accessible documentation and run-time signals (errors, warnings)
```